cypress/integration/inventory/moving-items.spec.js

```diff
@@ -0,0 +1,23 @@
+import TopMenu from '../../support/fragments/topMenu';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import testTypes from '../../support/dictionary/testTypes';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+
+describe('inventory: moving items', () => {
+  beforeEach('navigates to Inventory', () => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
```

Comment:
test has requirement about user permission. Need to prepare such user with minimal permissions around permission "Move items"